app/cdap/text/text-en.yaml

```diff
@@ -3108,6 +3108,7 @@ features:
     allTags: All Tags
     labelWithCount: Tags ({count})
     notags: No tags found. Click to add a new business tag.
+    notagsViewonly: No tags found.
```

Comment:
I understand that the previous one said notags but this should probably be pascalCase 'noTagsViewOnly'

Reply:
done